project.clj

```diff
@@ -50,7 +50,8 @@
                  [puppetlabs/ssl-utils "0.8.1"]
                  [puppetlabs/dujour-version-check "0.1.2" :exclusions [org.clojure/tools.logging]]
                  [puppetlabs/http-client "0.4.4"]
-                 [puppetlabs/comidi "0.3.1"]]
+                 [puppetlabs/comidi "0.3.1"]
+                 [org.apache.commons/commons-exec "1.3"]]
```

Comment:
Not a big deal but to keep these ordered this would probably go above, underneath `liberator` in that section.
Ones at the bottom are for our own PL libs.

More info: https://confluence.puppetlabs.com/pages/viewpage.action?pageId=48956980